src/views/AutogenView.vue

```diff
@@ -485,6 +485,10 @@ export default {
         }
       }
 
+      if (!['Admin'].includes(this.$store.getters.userInfo.roletype) && this.apiName === 'listTemplates' && this.selectedFilter !== 'self') {
```

Comment:
@Pearl1594 do we need to check for the selected filter; shouldn't we not display the order column irrespective of that? Maybe check against old UI?

Reply:
@rhtyd In the old UI - the order column is shown for all users , but the backend API prevents it from ordering them with the following exception: 
```
Domain Admin and regular users can modify only their own Public templates
```
Hence, to adhere to this - I've exposed the order column only for those that it owns

Reply:
@Pearl1594 so if an account owns templates, could they update and change order for their own (public) templates? Instead, we can just enforce that the order columns are shown only to the root admin.

Reply:
The API allows users and dom admins to change the order of their own public templates. However, I can restrict this only to Root admin